lib/datasource/npm/get.ts

```diff
@@ -35,7 +35,6 @@ export interface NpmDependency extends ReleaseResult {
   deprecationSource?: string;
   name: string;
   homepage: string;
-  latestVersion: string;
```

Comment:
I think this is used by internal tools

Reply:
It's only fetched for `npm`. Can you refactor to be `res.tags.latest` ?